torch/testing/_internal/common_methods_invocations.py

```diff
@@ -186,6 +207,8 @@ class OpInfo(object):
     def __init__(self,
                  name,  # the string name of the function
                  *,
+                 ref=None,  # An optional reference function that accepts NumPy arrays.
```

Comment:
"NumPy" -> "ndarrays (AKA "NumPy arrays")"

Reply:
Fixed in the recent commit, thanks!